spring-boot-modules/spring-boot-springdoc/src/main/java/com/baeldung/springdoc/tagorderexample/BooksController.java

```diff
@@ -0,0 +1,73 @@
+package com.baeldung.springdoc.tagorderexample;
+
+import java.util.ArrayList;
+import java.util.List;
+ 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import jakarta.validation.Valid;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+
+
+@RestController
+@Validated
+/**@OpenAPIDefinition(tags = {
+     @Tag(name = "books", description = "Books related tag"),
+     @Tag(name = "create", description = "Add book to inventory"),
+     @Tag(name = "delete", description = "Delete book from inventory"),
+     @Tag(name = "find", description = "Find book from inventory"),
+     @Tag(name = "update", description = "Update book in inventory"),
+     @Tag(name = "createBook", description = "Add book to inventory"),
+     @Tag(name = "deleteBook", description = "Delete book from inventory"),
+     @Tag(name = "findBook", description = "Find book from inventory"),
+     @Tag(name = "updateBook", description = "Update book in inventory")
+})**/
+@Tag(name = "books", description = "Books related tag")
+public class BooksController {
+
+    @Tag(name = "create")
+    @Tag(name = "books")
+    @Tag(name = "createBook")
+    @RequestMapping(path = "/book", method = RequestMethod.POST)
```

Comment:
On the methods, let's use the more recent Spring RestController annotations, @PostMapping, @GetMapping, and @DeleteMapping, instead of the older @RequestMapping.